controller/backup_target_controller.go

```diff
@@ -496,7 +496,7 @@ func (btc *BackupTargetController) syncBackupVolume(backupStoreBackupVolumeNames
 	// and delete the BackupVolume CR in the cluster
 	backupVolumesToDelete := clusterBackupVolumesSet.Difference(backupStoreBackupVolumes)
 	if count := backupVolumesToDelete.Len(); count > 0 {
-		log.Infof("Found %d backup volumes in the backup target that do not exist in the backup target and need to be deleted", count)
+		log.Infof("Found %d backup volumes in the backup target that do not exist in the cluster and need to be deleted from the cluster", count)
```

Comment:
Still not entirely non-confusing.  I might leave it at "need to be deleted."  I suppose it is really from the cluster's stored resources.  Anyway, better than it was.

Reply:
Agree that it's still a bit confusing, but I think it's really important to distinguish here that we are NOT deleting it from the backup target. That was the thing that scared me when I first saw it. It seemed to me that I hooked up a new cluster to an existing backup store and it started deleting my backups!